tests/expectations/compiler/finalize/closure_with_finalize_fail.out

```diff
@@ -22,7 +22,9 @@ Error [ETYC0372106]: An async function is not allowed to return a value.
     --> compiler-test:11:5
      |
   11 |     async function finalize_bar(a: u8, b: u8) -> u8 {
+     |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
For multi-line errors, the `^` feel somewhat distracting. Is it easy to not use them?